src/components/Panel/Panel.jsx

```diff
@@ -49,22 +49,39 @@ const Panel = React.createClass(createLucidComponentDefinition({
 	},
 
 	render: function() {
-		const {className, style, children, isGutterless} = this.props;
+		const {
+			className,
+			style,
+			children,
+			isGutterless
```

Comment:
Does _anyone_ else think this prop name is strange? It seems to add cognitive load to give a prop a "falsey" name that would have a `true` value as opposed to having a "truthy" name like `hasGutters`.